src/queryMetrics/clientSideMetrics.ts

```diff
@@ -0,0 +1,58 @@
+import { QueryMetricsUtils } from "./queryMetricsUtils";
+
+export class ClientSideMetrics {
+    private requestCharge: number;
+    constructor(requestCharge: number) {
+        // Constructor
+        if (!QueryMetricsUtils.isNumeric(requestCharge)) {
+            throw new Error("requestCharge is not a numeric type");
+        }
+
+        this.requestCharge = requestCharge;
+    }
+
+    /**
```

Comment:
Do we need these annotation comments still?

Reply:
Removed.